src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  margin: 0;
+  &__star {
+    background-image: url(../images/star.svg);
+    background-size: 80%;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+  &__star:last-child {
+    margin-right: 0;
+  }
+  @for $i from 1 through 5 {
+    &--#{$i} :nth-child(-n + #{$i}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
```

Comment:
The loop generates classes to handle the active stars, which is good. However, ensure that your HTML structure and JavaScript (if applicable) correctly apply these classes to display the correct number of active stars based on some condition or data.